corehq/apps/domain/static/domain/js/my_project_settings.js

```diff
@@ -15,11 +15,7 @@ hqDefine("domain/js/my_project_settings", [
         self.disableUpdateSettings = ko.observable(true);
 
         self.updateForm = function () {
-            if ($('#override_global_tz')[0].checked) {
```

Comment:
Hey @esoergel 
This change allows the user to unset/revoke the setting and then click update.
Currently if you uncheck the box, the update button is gone since `disableUpdateSettings` is set to true. 

https://github.com/dimagi/commcare-hq/blob/b5192db72f35b52d8527690a2a7ee22941aa668a/corehq/apps/domain/forms.py#L176-L183

Reply:
Ah I see - I misinterpreted this as controlling the visibility of the select box.  Nevermind then, this looks correct.